src/test/java/org/apache/cassandra/sidecar/routes/sstableuploads/BaseUploadsHandlerTest.java

```diff
@@ -222,4 +229,53 @@ public SidecarConfiguration configuration()
             return sidecarConfiguration;
         }
     }
+
+    static class TestCassandraAdapterDelegate extends CassandraAdapterDelegate
```

Comment:
can we call this `FakeCassandraAdapterDelegate` instead?

Reply:
But we already follow a pattern of calling such dummy classes Test<>. Example https://github.com/apache/cassandra-sidecar/blob/c72f2179143e7e031f247d3e8385a29c5e64c1c3/src/test/java/org/apache/cassandra/sidecar/routes/restore/BaseRestoreJobTests.java#L168